python/pyarrow/tests/test_array.py

```diff
@@ -3580,6 +3580,26 @@ def test_run_end_encoded_from_buffers():
                                            1, offset, children)
 
 
+def _generate_ree_array():
+    run_ends = [1, 3, 6]
+    values = [1, 2, 3]
+    ree_type = pa.run_end_encoded(pa.int32(), pa.int64())
+    return pa.RunEndEncodedArray.from_arrays(run_ends, values,
```

Comment:
Since we are not testing the constructor here (and now I merged your other PR), I think doing something simpler as `pa.array([1, 2, 2, 3, 3, 3], pa.run_end_encoded(pa.int32(), pa.int64()))` is also fine.